testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -130,7 +143,11 @@ void cleanup(void)
 		tst_resm(TWARN | TERRNO, "close(%s) failed", fname_mode1);
 	if (close(fd_mode2) == -1)
 		tst_resm(TWARN | TERRNO, "close(%s) failed", fname_mode2);
-	tst_rmdir();
+	// fallocate systemcall is failing on temporary directory created under /tmp
+	// Hence test directory is created in root filesystem.
+	remove(fname_mode1);
+      remove(fname_mode2);
+      rmdir(tempdir); // TODO: Revert back to original once git issue lsds/sgx-lkl#734 is fixed
```

Comment:
can you make this a url?

Reply:
@SeanTAllen : I didn't get. Could you please explain me

Reply:
Done